CIP-0839/README.md

```diff
@@ -0,0 +1,76 @@
+
+### The use of markdown in `description`
+
+While it is not strictly a requirement for implementation, the `description` field should be expected to include markdown formatting. An implementation *should* either strip the markdown altogether, or render it properly.
+
+### Result list
+In addition to the `title` and `description` fields, the `results` field must be present. This field is relevant in order to discern each of the options that a proposal can receive. This is useful in some governance implementations where, on-chain, options are associated in the datum with a number. In this case, the index into the array must associate to this mapping in the on-chain implementation. However, in governance systems where this isn't even a concern at all, it's still important to include this information. The `results` field is a list, each element being a record with the following required fields:
+| Field | Type  | Description | Can be IPFS |
+|--|--|--|--|
+| `name` | string | The name of the result  | **No** |
+| `description` | string | A description of what this result means | **Yes** |
+
+Similarly to before, the `name` field must be present, but the `description` can be an IPFS link.
+
+### Defining the creation transaction
+
+The exact point at which a proposal is created isn't super well-defined in all cases. So the requirements are just that it must be in some way:
+- unique: Two proposals can be distinguished in some way from one another. This can be through the datum, or a minting policy, or some other mechanism.
+- identifiable: Finding the transaction for the creation of a proposal doesn't require complex datum-delving.
+- governance-associated: Each governance instance must differ in some way from one another.
+
+Implementing these requirements can be done through additional metadata fields as well.
+
+### Extension
+
+Implementations should not be rigid in interpreting this tag's contents, as various extensions may be applied over top of this. For example, one use case could be applying categories to proposals, or specific milestones. Due to the nature of the standard, these extensions necessarily are optional, and can help hydrate the information presented to the users.
+
+## Rationale: how does this CIP achieve its goals?
+
+The decisions made here are mostly a retroactive generalization of the approach taken in the implementation of the frontend in [Agora]'s [first instance](https://app.liqwid.finance/governance). This of course means that it is at risk of being too specific for this use case. However, I believe it is sufficiently general for any other on-chain governance implementation in the future to adopt, and due to Agora's open source nature, it should not exactly pose a threat to anything that may compete.
+
+Additionally, since this is a metadata proposal with very little burden of implementation, it should not clash with any existing CIPs nor any in the future.
+
+[Agora]: https://github.com/Liqwid-Labs/agora/
+
```

Comment:
```suggestion
## Path to Active
```
Please provide & fill out a Path to Active section: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#path-to-active

Also if Agora is a separate entity than Liqwid please elaborate on their respective responsibilities here: and if Liquid is actually the implementing agency (or piloting the implementation) then please indicate so in the preamble at the top. 🤓